bestax-migrate/scripts/validate-corpus.mjs

```
@@ -0,0 +1,146 @@
+  // package-import form real consumer code uses.
+  const source = fs
+    .readFileSync(file, 'utf8')
+    .replace(/from '(\.\.\/)*\.\.(\/index)?'/g, "from 'react-bulma-components'")
+    .replace(/from "(\.\.\/)*\.\.(\/index)?"/g, 'from "react-bulma-components"')
+    .replace(
+      /import (\w+) from '\.(?:\/[\w-]+)?';/g,
+      "import { $1 } from 'react-bulma-components';"
+    );
+
+  const todos = [];
+  let output = null;
```

ESLint

[error] The value assigned to 'output' is not used in subsequent statements.

(no-useless-assignment)